src/Kafka/ManageKafkaPermissions/components/ResourcePrefix.tsx

```diff
@@ -0,0 +1,38 @@
+import { FormGroup, TextInput, ValidatedOptions } from "@patternfly/react-core";
+import { useTranslation } from "react-i18next";
+
+export type ResourcePrefixProps = {
+  value: string | undefined;
+  onChangeValue: (value: string | undefined) => void;
+  invalid: boolean;
+};
+
+export const ResourcePrefix: React.VFC<ResourcePrefixProps> = ({
+  value,
+  onChangeValue,
+  invalid,
+}) => {
+  const { t } = useTranslation(["manage-kafka-permissions"]);
+
+  const onPrefixValueChange = (value: string | undefined) => {
+    onChangeValue(value);
+  };
+
+  return (
+    <FormGroup
+      validated={invalid ? ValidatedOptions.error : ValidatedOptions.default}
+      helperTextInvalid={t("common:required")}
+      fieldId={"resource-prefix-input"}
+    >
+      <TextInput
+        placeholder={t("resourcePrefix.placeholder_text")}
+        validated={invalid ? ValidatedOptions.error : ValidatedOptions.default}
+        value={value}
+        type="text"
+        onChange={onPrefixValueChange}
+        aria-label="Prefix value text input"
```

Comment:
Whenever `aria-label` is defined it should always use a translated string. 

An aria-label also doesn't need to describe the thing—that's like putting the word "button" inside of a button 😉 Usually it just repeats the visible text that displays adjacent to the input.

Looking at the current implementation, this attribute doesn't provide user-friendly text. 

In this case the aria-label I would recommend is something like this: 
`"{selected resource type} prefix"` so that it would say "Topic prefix" or "Consumer group prefix", depending on the resource type they selected in the first column.

<img width="619" alt="image" src="https://user-images.githubusercontent.com/21063328/158888880-633c226c-54bc-4d33-aee8-e1d6dc038c97.png">